index.bs

```diff
@@ -1861,39 +1861,44 @@ Content of a Charter</h3>
 	who represent the same Member organization or group of <a href="#MemberRelated">related Members</a>).
 
 <h3 id="CharterReview">
-Advisory Committee Review of a Charter</h3>
+Charter Review and Approval</h3>
 
-	The [=Team=] <em class="rfc2119">must</em> solicit <a href="#ACReview">Advisory Committee review</a>
-	of every new or substantively modified Working Group or Interest Group charter,
-	except for either:
-	* a charter extension
-	* substantive changes to a charter that do not affect the way the group functions in any significant way.
+<h4 id="charter-major-changes">
+New Charters and Major Changes</h4>
 
-	The review period <em class="rfc2119">must</em> be at least 28 days.
-	The following are examples of substantive changes that would not require an [=Advisory Committee Review=]:
-	the addition of an in-scope deliverable,
-	a change of [=Team Contact=],
-	or a change of [=Chair=].
-	Such changes must nonetheless be announced
-	to the [=Advisory Committee=]
-	and to <a href="#wgparticipant"> participants in the Working</a> or <a href="#igparticipant">in the Interest Group</a>,
-	and a rationale must be provided.
+	Any new [=charter=] (including re-chartering of existing [=chartered groups=])
+	and most substantive changes to an already-approved [=charter=]
+	of a [=Working Group=] or [=Interest Group=]
+	<em class=rfc2119>must</em> be approved by a [=W3C Decision=] following an [=AC Review=] of that [=charter=].
```

Comment:
I tend to prefer whichever markup pattern is shortest without those that are valid HTML. HTML allows this pattern, so that's why I go for it when possible.
https://html.spec.whatwg.org/multipage/syntax.html#unquoted

For this particular markup pattern, I don't expect the class attribute to be expanded with additional classes, so the `"` wouldn't even help preparing for their addition, so I'm happy to save a little bit of line noise, even if it's minor.

With that said, I wouldn't object using quotes here if others have a preference the other way around, it's only a minor preference of mine. And as you say, they get added when the output is generated anyway, so that makes it matter even less.

Reply:
@fantasai, as co-editor, agrees with the above logic, so closing this comment. @TallTed, if you really care, you could open a separate issue. My expectation is that we'll get consistent eventually (since both editors agree on the prefered approach), but I don't think it's worth making a giant edit to make it consistent in one go.